client/src/components/DayView/Calendar.js

```diff
@@ -128,6 +128,10 @@ class Calendar extends React.Component {
     this.showModal();
   };
 
+  addnewEvent = (events, event) => {
+    this.setState({ events: [...events, event] });
+  };
+
```

Comment:
i think its better to refetch the data from the server..

Reply:
so we know we have the real data, not something not verified by the server

Reply:
It's verified, as the booking process result goes well.
I think we don't have to fetch data from the server as we have it, especially that the addition done after server response is OK

Reply:
When i have many events on the same day, all of them disappear when you add a new event, I'm sure this is not intended and this is the reason why I told you to just re-call the fetch function
you can do something like 
```js
this.componentDidMount();
```
this calls the component did mount again, because its just a function in our class